test/DripsHub.t.sol

```diff
@@ -378,16 +349,28 @@ contract DripsHubTest is Test {
         assertEq(collectable(forUser), expected, "Invalid collectable");
     }
 
+    function totalBalance() internal view returns (uint256) {
+        return dripsHub.totalBalance(erc20);
+    }
+
     function assertTotalBalance(uint256 expected) internal {
-        assertEq(dripsHub.totalBalance(erc20), expected, "Invalid total balance");
+        assertEq(totalBalance(), expected, "Invalid total balance");
     }
 
     function balance() internal view returns (uint256) {
         return erc20.balanceOf(driver);
     }
 
     function assertBalance(uint256 expected) internal {
-        assertEq(erc20.balanceOf(driver), expected, "Invalid balance");
+        assertEq(balance(), expected, "Invalid balance");
+    }
+
+    function dripsHubBalance() internal view returns (uint256) {
```

Comment:
In general good design of all the tests. That didn't result in 100 changes of replacing `balanceOf` reserve with `balanceOf` DripsHub.